### ISSUE TITLE
chess|new problem

### ISSUE THREAD

**Issue by michaelprimo** [timburgan/timburgan#9991]
cc @timburgan If someone try to reset the game with chess|new while the game is still on only the moves will be resetted and not the chessboard, blocking people to play the game.

**Comment by timburgan** [comment]
Thanks for letting me know

**Comment by deathstar1** [comment]
yup , i noticed  the same today  when i  chess|new to play my favorite game on github and found the same issue . Thanks.

**Comment by michaelprimo** [comment]
I think the same ^_^ Merry Christmas to you all!
